region/norfair/east.json

```diff
@@ -1971,6 +1971,12 @@
           "homeNodes": [ 8 ]
         }
       ],
+      "reusableRoomwideNotable": [
+        {
+          "name": "Bubble Mountain Damage Boost",
+          "note": "Damage Boost off a Waver to cross the gap."
```

Comment:
Raises the question of how identical `note`s are to be handled.  Notable strats shouldn't truely be identical (maybe).  A LtoR setup here would wait a while for a waver to come back to the door to be in a good position.  A RtoL setup either rushes to meet the waver, or waits for it to bounce off the right side wall and come at Samus from behind.

This note would be the user facing description.
And the full strat description will have a note that only appears in the spoiler log.
That being the case it seems fine for notes to be the same.

This description sounds pretty boring though so maybe this dboost shouldn't be notable.  The action required is just to wait for the waver to be overhead and jump into it and damage boost.

Reply:
The `note` in the `reusableRoomwideNotable` will show up when selecting which notable strats the player is interested in being in logic, so it will be more generic and simplified. The `note`s in the strats themselves can be more complex and describe the nuances of the particular strat. (All notable strat `note`s need to be worked on soon.)

As for this strat in particular. I kept it in here because it has an unforgiving fall below it. Similar to the PCJR Grapple, Early Supers Mockball, etc

Reply:
I'm thinking there is some need to describe the situation as we're detached from room nodes.  Wording should be more proper.  What makes this strat notable should be mentioned in one of the notes, but could be on either.

"Crossing between the topmost doors of Bubble Mountain by damage boosting using a Waver."
"Using a Waver damage boost to propel Samus across the top of the Bubble Mountain room."
"Avoid falling into the center of Bubble Mountain by using a damage boost to directly cross the top"

Reply:
I think this is a good suggestion to do eventually. Most of the notable strats notes could use some work, as well as many of the other notes. I've edited the Bubble Mountain note, but I'm thinking this will be a project for a future pass through.